xclim/ensembles/_base.py

```diff
@@ -375,10 +387,19 @@ def _ens_align_datasets(
     if isinstance(datasets, str):
         datasets = glob(datasets)
 
+    if mf_flag != "UNSET":
+        warnings.warn(
+            "The `mf_flag` argument is being deprecated in favour of `multifile` in `_ens_align_datasets()`. "
+            "This change will be made effective from `xclim>=0.43.0`. Please update your scripts accordingly",
+            DeprecationWarning,
```

Comment:
```suggestion
            FutureWarning,
```
Future Warnings are visible to the user, while Deprecation Warnings are not. It confuses me everythime... https://docs.python.org/3/library/warnings.html?highlight=warnings#warning-categories

Reply:
Dang. That's good to know!